<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TEACHING ROOM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/appFace/systray</normaloff>:/appFace/systray</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="logo">
             <property name="minimumSize">
              <size>
               <width>270</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/appFace/loginicon);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QToolButton" name="weiBroadcastBtn">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/appFace/weibroabcast</normaloff>:/appFace/weibroabcast</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="messageBoxBtn">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>12</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/appFace/messagebox</normaloff>:/appFace/messagebox</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="student">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <attribute name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/appFace/student</normaloff>:/appFace/student</iconset>
           </attribute>
           <attribute name="title">
            <string/>
           </attribute>
           <attribute name="toolTip">
            <string>学生列表</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="refreshBtn">
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>刷新学生列表</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/appFace/refresh</normaloff>:/appFace/refresh</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="adlabel0">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="classManageBtn">
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>班级管理</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/appFace/classmm</normaloff>:/appFace/classmm</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTreeWidget" name="studenTree">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/appFace/bg);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::HLine</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="lineWidth">
                 <number>2</number>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="alternatingRowColors">
                 <bool>false</bool>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="iconSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="autoExpandDelay">
                 <number>0</number>
                </property>
                <property name="indentation">
                 <number>10</number>
                </property>
                <property name="uniformRowHeights">
                 <bool>false</bool>
                </property>
                <property name="allColumnsShowFocus">
                 <bool>false</bool>
                </property>
                <attribute name="headerVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerHighlightSections">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="questions">
           <attribute name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/appFace/questions</normaloff>:/appFace/questions</iconset>
           </attribute>
           <attribute name="title">
            <string/>
           </attribute>
           <attribute name="toolTip">
            <string>问题列表</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QToolButton" name="createQuestionBtn">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>点击新建一个问题</string>
                  </property>
                  <property name="text">
                   <string>新建问题</string>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/appFace/createquestion</normaloff>:/appFace/createquestion</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="refreshQuestionBtn">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>点击刷新问题列表</string>
                  </property>
                  <property name="text">
                   <string>刷新问题列表</string>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/appFace/refresh</normaloff>:/appFace/refresh</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTreeWidget" name="questionSetList">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/appFace/bg);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::HLine</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="lineWidth">
                 <number>2</number>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="indentation">
                 <number>10</number>
                </property>
                <attribute name="headerVisible">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>1</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="broadcast">
           <attribute name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/appFace/broadcasts</normaloff>:/appFace/broadcasts</iconset>
           </attribute>
           <attribute name="title">
            <string/>
           </attribute>
           <attribute name="toolTip">
            <string>广播列表</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QListWidget" name="broadCastList"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="functionBoxBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/appFace/functionbox</normaloff>:/appFace/functionbox</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件[&amp;F]</string>
    </property>
    <addaction name="logoutAction"/>
    <addaction name="quitAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助[&amp;H]</string>
    </property>
    <addaction name="helpCenterAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具[&amp;T]</string>
    </property>
    <addaction name="menuChangeAction"/>
    <addaction name="changePasswordAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menu_T"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="logoutAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/appFace/logout</normaloff>:/appFace/logout</iconset>
   </property>
   <property name="text">
    <string>注销[&amp;N]</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/appFace/quit</normaloff>:/appFace/quit</iconset>
   </property>
   <property name="text">
    <string>退出[&amp;X]</string>
   </property>
  </action>
  <action name="helpCenterAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/appFace/helpcenter</normaloff>:/appFace/helpcenter</iconset>
   </property>
   <property name="text">
    <string>帮助中心[&amp;H]</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/appFace/about</normaloff>:/appFace/about</iconset>
   </property>
   <property name="text">
    <string>关于[&amp;A]</string>
   </property>
  </action>
  <action name="menuChangeAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/appFace/hidemenu</normaloff>:/appFace/hidemenu</iconset>
   </property>
   <property name="text">
    <string>隐藏菜单[ALT]</string>
   </property>
   <property name="toolTip">
    <string>隐藏菜单</string>
   </property>
  </action>
  <action name="changePasswordAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/appFace/changepassword</normaloff>:/appFace/changepassword</iconset>
   </property>
   <property name="text">
    <string>修改密码[&amp;C]</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>weiBroadcastBtn</tabstop>
  <tabstop>messageBoxBtn</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
